feature-libs/cart/base/core/facade/active-cart.service.spec.ts

```diff
@@ -326,6 +372,15 @@ describe('ActiveCartService', () => {
       });
     });
 
+    it('should be called if user logged in with code flow', () => {
```

Comment:
Proposal: `should be called if user is logged in with code flow`

Reply:
Changed